docs/standard/microservices-architecture/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md

```diff
@@ -29,21 +29,16 @@ The concepts of a cluster and a scheduler are closely related, so the products p
 
 ## Software platforms for container clustering, orchestration, and scheduling
 
-### Kubernetes
-
-![Kubernetes logo](./media/image24.png)
-
-> [*Kubernetes*](https://kubernetes.io/) is an open-source product that provides functionality that ranges from cluster infrastructure and container scheduling to orchestrating capabilities. It lets you automate deployment, scaling, and operations of application containers across clusters of hosts.
->
-> *Kubernetes* provides a container-centric infrastructure that groups application containers into logical units for easy management and discovery.
->
-> *Kubernetes* is mature in Linux, less mature in Windows.
-
-### Azure Kubernetes Service (AKS)
-
-![Azure Kubernetes Service logo](./media/image41.png)
-
-> [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/services/kubernetes-service/) is a managed Kubernetes container orchestration service in Azure that simplifies Kubernetes cluster’s management, deployment, and operations.
+|     |   |
```

Comment:
I tried to fix the table formatting, but the table continues to render incorrectly.

Reply:
The change I made should fix it but I can't see what's the advantage of putting that information on a table

Reply:
Hi @rpetrusha, @mairaw.

That's the original e-book format and it uses less space, I was using [docs directions](https://docs.microsoft.com/contribute/markdown-reference#tables) to have a headerless table, although now I realize the header separator I used is not exactly as the guide, but it rendered just as expected with Pandoc, Could you paste the image to see how it's being rendered?

Anyway I just found that Markdig supports [Grid Tables](https://github.com/lunet-io/markdig/blob/master/src/Markdig.Tests/Specs/GridTableSpecs.md) from Pandoc so it would make it MUCH easier to format tables that render fine in the e-books.

Could you check if the sample in the linked page renders fine?